CHANGES.rst

```diff
@@ -12,6 +12,7 @@ Enhancements
 * The order of the context name resolution (and ``$ContextPath``) switched putting ``"System`"`` before ``"Global`"`.
 * In assignment to messages associated with symbols, the attribute ``Protected`` is not having into account, folliwing the standard in WMA. With this and the above change, Combinatorical 2.0 works as written.
 * ``Share[]`` performs an explicit call to the Python garbage collection and returns the amount of memory free.
+* Improving the compatibility of ``TeXForm`` and ``MathMLForm`` outputs with WMA.
```

Comment:
If you are up to it, maybe a little bit about how the compatibility was changed. I looked hard to find _where_ the `mtext` tag was change to the `mn` tag.  I guess what is going on is that we are doing better about detecting the type and using the OO behavior. So that `mn` tag` code was already there, but it is now just getting used properly. 

I see that quotes in TeXForm get changed as well.  Personally, I think it would be helpful as a user to know that in TeXForm some quotes may disappear and that in MathForm some mtext tags may become mn tags.  But I realize this is a matter of taste.

Reply:
What is puzzling to me is to see how and where to document these changes. There is a rationale for why a number in StandardForm / OutputForm must be tagged with ``<mn>`` in MathMLForm, while in InputForm are tagged as 
``<mtext>``: InputForm is supposed to represent how an expression must be introduced, while StandardForm/OutputForm
tries to represent what it actually is: a number.

In the case of quotes in `TeXForm[InputForm[..]]`,  the problem is that I am not sure that the WMA output is right. However, if we want to be able to load WMA packages in Mathics, we need to keep the maximum compatibility. In any case, now it is more clear than before what we need to change in order to have the other behavior.